cpp/src/arrow/dataset/file_parquet_encryption_test.cc

```diff
@@ -188,8 +286,13 @@ TEST_F(DatasetEncryptionTest, WriteReadDatasetWithEncryption) {
   AssertTablesEqual(*combined_table, *table_);
 }
 
-// Read a single parquet file with and without decryption properties.
-TEST_F(DatasetEncryptionTest, ReadSingleFile) {
+void read_single_file(
+    std::shared_ptr<fs::FileSystem> file_system_, std::shared_ptr<Table> table_,
```

Comment:
Local variables and function arguments should not have a trailing underscore. The trailing underscore is for class member variables.